sig-and-wg/sig-core/proposals/introducing_knative_brokers.md

```diff
@@ -0,0 +1,88 @@
+# Integration of the Knative-based function controller
+
+## Introduction
+
+TBD
+
+## Features
+
+### Current Features
+
+- As a developer, I want to publish events to Kyma from outside the cluster.
+- As a developer, I want to publish events to Kyma from inside the cluster.
+- As a developer, I want to subscribe to events in Kyma from a specific source with a specific event-type.
+- As an admin, I want to enable or disable events in a specific namespace.
+
+### New features
```

Comment:
This is not a new feature. Right now, the cm `default-ch-webhook` exists which enables this functionality.